src/coreclr/vm/method.cpp

```diff
@@ -253,6 +253,30 @@ HRESULT MethodDesc::SetMethodDescVersionState(PTR_MethodDescVersioningState stat
     return S_OK;
 }
 
+#ifdef FEATURE_INTERPRETER
+// Set the call stub for the interpreter to JIT/AOT calls
+// Returns true if the current call set the stub, false if it was already set
+bool MethodDesc::SetCallStub(CallStubHeader *pHeader)
+{
+    LIMITED_METHOD_CONTRACT;
```

Comment:
```suggestion
    STANDARD_VM_CONTRACT;
```
This throws now, so it is not limited contract anymore. (This change requires making the GCX_PREEMP scope larger.)